frontend/src/v5/store/dialogs/dialogs.redux.ts

```diff
@@ -32,6 +33,11 @@ export const { Types: DialogsTypes, Creators: DialogsActions } = createActions({
 }, { prefix: 'MODALS/' }) as { Types: Constants<IDialogsActionCreators>; Creators: IDialogsActionCreators };
 
 export const openHandler = (state, { modalType, props }: OpenAction) => {
+	// avoid opening 2+ redirect modals
```

Comment:
This causes a crash when a modal doesn't contain an error. For instance the confirm delete, upload files, and revision still processing info modals.